tests/src/test/java/alluxio/client/fs/ConcurrentFileSystemMasterLoadMetadataTest.java

```diff
@@ -37,20 +37,7 @@
 
   @Rule
   public LocalAlluxioClusterResource mLocalAlluxioClusterResource =
-<<<<<<< HEAD
       new LocalAlluxioClusterResource.Builder().build();
-=======
-      new LocalAlluxioClusterResource.Builder()
-          .setProperty(PropertyKey.USER_FILE_MASTER_CLIENT_POOL_SIZE_MAX, CONCURRENCY_FACTOR)
```

Comment:
should these be set?

Reply:
There was no `CONCURRENCY_FACTOR` variable defined in the file - so whatever change regarding this test that was made wasn't cherry-picked, so I left it out.